new/src/components/button/button.scss

```diff
@@ -1,4 +1,4 @@
-.button {
+button[class^="button"] {
```

Comment:
Any bug if we leave it as `.button`?

Reply:
I faced issue due to a redundant css property defined in another file 🤣 
Changed back to using class name selectors, and made the discord button inherit both button classes' property